variables.tf

```diff
@@ -86,3 +86,35 @@ variable "cbr_rules" {
   default     = []
   # Validation happens in the rule module
 }
+
+##############################################################################
+# Event Notification
+##############################################################################
+
+variable "skip_en_iam_authorization_policy" {
+  type        = bool
+  description = "Set to true to skip the creation of an IAM authorization policy that permits all Secrets Manager instances (scoped to the resource group) an 'Event Source Manager' role to the given Event Notifications instance passed in the `existing_en_instance_crn` input variable. In addition, no policy is created if `enable_event_notification` is set to false."
+  default     = false
+}
+
+variable "enable_event_notification" {
+  type        = bool
+  default     = false
+  description = "Set this to true to enable lifecycle notifications for your Secrets Manager instance by connecting an Event Notifications service. When setting this to true, a value must be passed for `existing_en_instance_crn` variable."
+}
+
+variable "existing_en_instance_crn" {
+  type        = string
+  default     = null
+  description = "The CRN of the Event Notifications service to enable lifecycle notifications for your Secrets Manager instance."
+}
+
+variable "endpoint_type" {
```

Comment:
To avoid confusion I would go with `event_notification_endpoint_type`, or even `en_endpoint_type`